harvester/sources/tests/extraction/test_edurep.py

```diff
@@ -0,0 +1,266 @@
+    #     for ix, data in enumerate(zip(self.json_data, self.xml_data)):
+    #         json_docu, xml_docu = data
+    #         self.assertEqual(set(json_docu["lom_educational_levels"]), set(xml_docu["lom_educational_levels"]),
+    #                          f"Documents are unequal at index {ix} \n JSON: \n {json_docu} \n\n XML: \n {xml_docu}")
+
+
+class TestGetHarvestSeedsEdurep(TestCase):
+
+    begin_of_time = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.begin_of_time = make_aware(datetime(year=1970, month=1, day=1))
+        EdurepJsonSearchResourceFactory.create_common_responses()
+        cls.seeds = get_harvest_seeds(Repositories.EDUREP_JSONSEARCH, SET_SPECIFICATION, cls.begin_of_time)
+
+    def test_get_id(self):
+        seeds = self.seeds
+        self.assertEqual(seeds[0]["external_id"], "jsonld-from-lom:wikiwijsmaken:41156")
+
+    def test_state_education_level(self):
+        seeds = self.seeds
+
+        for i in range(5):
+            state = seeds[i]["state"]
+            lowest_educational_level = seeds[i]["lowest_educational_level"]
+            if lowest_educational_level < 2:
+                self.assertEqual(state, "inactive")
+            else:
+                self.assertEqual(state, "active")
+
+    def test_lowest_education_level(self):
+        seeds = self.seeds
+
+        self.assertEqual(seeds[0]["lowest_educational_level"], 1,
+                         "Expected file with MBO to have lowest educational level 1")
+        self.assertEqual(seeds[1]["lowest_educational_level"], 1,
+                         "Expected file with MBO to have lowest educational level 1")
+        self.assertEqual(seeds[2]["lowest_educational_level"], 2,
+                         "Expected file with HBO to have lowest educational level 2")
+
+    def test_keywords(self):
+        seeds = self.seeds
+        self.assertTrue("asperge" in seeds[0]["keywords"])
+        self.assertTrue("wortelen" in seeds[0]["keywords"])
+        self.assertTrue("vollegrondsgroenten" in seeds[0]["keywords"])
+        self.assertTrue("sperziebonen" in seeds[0]["keywords"])
+
+    def test_copyright_description(self):
+        seeds = self.seeds
+        self.assertEqual(seeds[2]["copyright_description"], "cc-by-sa-30")
+        self.assertEqual(seeds[1]["copyright_description"], "cc-by-30")
+        self.assertEqual(seeds[0]["copyright_description"], "cc-by-30")
+        self.assertEqual(seeds[3]["copyright_description"], "http://creativecommons.org/licenses/by-nc-sa/4.0/")
+        self.assertEqual(seeds[4]["copyright_description"], "onbekend")
+        self.assertEqual(seeds[5]["copyright_description"], "cc-by-nc-nd-30")
+
+    def test_copyright(self):
+        seeds = self.seeds
+        self.assertEqual(seeds[2]["copyright"], "https://creativecommons.org/licenses/by-sa/3.0/nl/")
+        self.assertEqual(seeds[3]["copyright"], "cc-by-nc-sa-40")
+        self.assertEqual(seeds[4]["copyright"], "yes")
```

Comment:
@fako not certain if this is the correct output for the copyright value. This and the copyright description gave me a lot of headaches. Important to review.